node/node_bridge_test.go

```diff
@@ -154,16 +48,3 @@ func TestBridge_WithMockedCoreClient(t *testing.T) {
 	err = node.Stop(ctx)
 	require.NoError(t, err)
 }
-
-func TestBridge_WithMutualPeers(t *testing.T) {
```

Comment:
There is the same test for a light client. There is zero value in testing options for both node types as the code for them is the same. I would just use light client only for testing options.